website/docs/cloud-docs/users-teams-organizations/single-sign-on/linking-user-account.mdx

```diff
@@ -16,7 +16,7 @@ The first time you use SSO to log in to an organization, HCP Terraform links tha
 
 When HCP Terraform does not recognize the email address associated with your identity provider, it asks if you want to create a new user account with that email address. You can choose one of the following actions:
 
-- **Create a new account:** HCP Terraform automatically links your SSO identity to this new account after creation.
+- **Create a new account:** [HCP Terraform](https://app.terraform.io/) automatically links your SSO identity to this new account after creation.
```

Comment:
not sure about the link here since there's nothing actionable

Reply:
Sounds good! I'll remove
```suggestion
- **Create a new account:** HCP Terraform automatically links your SSO identity to this new account after creation.
```